.travis.yml

```diff
@@ -408,21 +408,20 @@ jobs:
       - KEPTN_INSTALLATION_TYPE=FULL
     <<: *gke_full # use template
 
-  - stage: Test GKE Full (--platform=gke)
-    if: branch = master AND type = cron # run for cron
-    env:
-      - GKE_VERSION=1.15
```

Comment:
Just curious, did you disable 1.15 tests to save time/money or is there any other reason?

Reply:
To save time, money and resources.

1.14 and 1.15 are very similar, so there is no benefit from testing both. 1.14 however is the oldest version we are still supporting.